client/components/callout.js

```diff
@@ -3,19 +3,19 @@ const cta = require('./cta-btn')
 
 module.exports = function (state, emit) {
   return html`
-    <div class="mt7-ns mt5 mw8 center tl flex flex-row-ns flex-column w-100">
+    <div class="mt5-ns mt5 mw8 center tl flex flex-row-ns flex-column w-100">
       <div class="flex w-100 w-third-ns bg-white dat-shadow mv4-ns mv2 mh3-l mh1-ns pa3">
         <div class="flex-column flex">
           <div class="mt3 w3 h3 center">
             <svg><use xlink:href="#daticon-star-dat"/></svg>
           </div>
-          <h4 class="f4 mb0 tc">datBase</h4>
+          <h4 class="f4 mb0 tc">Dat Base</h4>
           <p class="tc mb0 f7 ttu dat-neutral-70">
-            Coming soon!
+            A Public Dat Peer
```

Comment:
changed this to `dat for researchers` because its not a public peer yet and I kinda said we wouldn't ship that yet in the last blog post =)